react/yarn.lock

```diff
@@ -3575,9 +3575,13 @@ posix-character-classes@^0.1.0:
   integrity sha1-AerA/jta9xoqbAL+q7jB/vfgDqs=
 
 "posthog-js@link:..":
-  version "1.83.1"
-  dependencies:
-    fflate "^0.4.1"
+  version "0.0.0"
+  uid ""
+
+preact@^10.19.3:
```

Comment:
I tested it quickly, the only difference in bundle size is when we actually use JSX in `surveys.tsx` (which we don't in this PR currently). Then the output `surveys.js` goes from 48 Kb to 57 Kb, because we do actually bundle Preact – but it's relatively lightweight, so not a massive jump considering the benefits.

Also just made sure, we only load `surveys.js` with `loadScript()` when there actually are surveys to show, so the existing bundling setup actually works already. Which is great, well done setting this up 😄